jsdoc.md

```diff
@@ -0,0 +1,12 @@
+<a name="FetchDropboxAsset"></a>
```

Comment:
I thought we just have jsdoc for services, just for developers reference and not to be put in the readme. We already have an example in Library jsdoc.